### PR TITLE
Add custom thumbnail property (rename old one to thumbnail-size)

### PR DESCRIPTION
**Pull request by ellell**
Type: Major
Review: @kesla @iefserge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ Function to run when video is clicked on
 #### `opened=[boolean]`
 If false, shows a video preview using the youtube thumbnail and if true opens and plays the video.
 
-#### `thumbnail=[string]` - Optional
+### `thumbnail=[string]` - Optional
+Set custom thumbnail image, this will override the standard youtube thumbnail.
+
+#### `thumbnail-size=[string]` - Optional
```
Comment (kesla) [inline README.md]: let's go with thumbnailSize

File: test.js [hunk 2]
```diff
@@ -23,6 +23,46 @@ test('YoutubeVideo initial state', function (t) {
   t.end();
 });
 
+test('YoutubeVideo thumbnail-size', function (t) {
+  var html = renderString(tree(YoutubeVideo.render({
+    props: {
+      'youtube-id': 'YoB8t0B4jx4',
```
Comment (kesla) [inline test.js]: if we're going to break compatibility, perhaps rename to `youtubeId`?

**Comment by kesla** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (ellell) [inline README.md]: @kesla ok, lets also change youtube-id then while we're at it! :)

**Comment by kesla** [comment]
or, I had 2 small comments, but rest looks great

File: test.js [hunk 3]
```diff
@@ -23,10 +23,50 @@ test('YoutubeVideo initial state', function (t) {
...
+    </div>`);
+  t.end();
+});
+
+test('YoutubeVideo thumbnail', function (t) {
+  var html = renderString(tree(YoutubeVideo.render({
+    props: {
+      youtubeId: 'YoB8t0B4jx4',
+      opened: false,
+      'thumbnail': 'http://custom-image.jpg'
```
Comment (iefserge) [inline test.js]: unnecessary quotes

**Pull request merged**